test/external/CMakeLists.txt

```diff
@@ -2,5 +2,31 @@
 # Add tests based on external repositories
 #
 
+include(FetchContent)
+
+FetchContent_Declare(
+    ringtest
+    GIT_REPOSITORY https://github.com/neuronsimulator/ringtest
```

Comment:
For test repositories we should have a CMake variable, i.e: 
```
-DNRN_EXTERNAL_TEST_REPOS="https://github.com/neuronsimulator/ringtest; https://github.com/neuronsimulator/testcorenrn:some_branch"
```
if branch/commit/tag not specified, then we take master. For example above: 
* ringtest -> `master` (head of default branch to be specific)
* testcorenrn -> `some_branch`

Reply:
Discussed offline: will come back to `-DNRN_EXTERNAL_TEST_REPOS` at a later time.